src/components/Search/Search.scss

```diff
@@ -139,4 +139,12 @@
       margin: 0;
     }
   }
+
+  &.tm-w-xs,
+  &.tm-w-s {
+    .react-autosuggest__input {
+      font-size: 16px;
+      padding: 12px 15px 9px 15px;
```

Comment:
try to find a padding that work for large and small screen if possible.

Reply:
Not related to this PR but also I see the text oes under the cross and the red button.

Reply:
The font-size change on mobile makes the input bigger, I have to adapt the padding, because we don't want to set the font-size to 16px on normal screens.

Reply:
I fixed the text going under the clear & search button additionally.